crates/youki/src/logger.rs

```diff
@@ -19,12 +19,13 @@ const LOG_FORMAT_JSON: &str = "json";
 /// Initialize the logger, must be called before accessing the logger
 /// Multiple parts might call this at once, but the actual initialization
 /// is done only once due to use of OnceCell
-pub fn init(log_format: Option<String>, log_file: Option<PathBuf>) -> Result<()> {
+pub fn init(log_debug_flag: bool, log_file: Option<PathBuf>, log_format: Option<String>) -> Result<()> {
     let formatter = match log_format.as_deref() {
         None | Some(LOG_FORMAT_TEXT) => text_write,
         Some(LOG_FORMAT_JSON) => json_write,
         Some(unknown) => bail!("unknown log format: {}", unknown),
     };
+    let loglevel = if log_debug_flag { "debug" } else { "warn" };
```

Comment:
Do you use `loglevel` anywhere? I think you need to use this flag, perhaps around line 41?

Reply:
> Do you use loglevel anywhere? I think you need to use this flag, perhaps around line 41?  

I don't know what I should do after this, can you give me some tips?

Reply:
You need to pass the log level to the logger builder. It has a filter_level method that should do the trick. You should set the log level to debug if the debug flag is specified, otherwise you need to check the youki log environment variable. If both are not set, you should fall back to the default log level.

Reply:
> You need to pass the log level to the logger builder. It has a filter_level method that should do the trick. You should set the log level to debug if the debug flag is specified, otherwise you need to check the youki log environment variable. If both are not set, you should fall back to the default log level.

![image](https://user-images.githubusercontent.com/52807407/141234569-fb4c3a13-b877-4bde-8e57-7c1fbf4b5caa.png)
like this? Or should I make some other changes?

Reply:
```suggestion
    let loglevel = if log_debug_flag {
        "debug"
    } else {
        DEFAULT_LOG_LEVEL
    };
```

and set a variable `loglevel` instead of `DEFAULT_LOG_LEVEL` at L41.
How about this?

Reply:
hmm